apps/web/components/ui/HeroCarousel/types.ts

```diff
@@ -11,6 +11,7 @@ export type Sizes = {
 
 export type HeroItem = {
   image: string;
+  alt: string;
```

Comment:
**question**: When we bring back the different image sizes, our `image` property will look like this, right?

```
image: {
  lg: string,
  md: string,
  sm: string
}
```
Should `alt` also become part of `image` then?